### DISCUSSION TITLE
Different ways to use terminal on windows school computers when it is blocked

### DISCUSSION THREAD

**Discussion by Inglan2** [3kh0/ext-remover#1093] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
1. Windows key + R then `cmd` then ok (prob wont work but put here anyway)
2. Windows key + R then `powershell` then ok
3. Windows key + R then `ftp` then ok. To run commands type `!` and then the command (for example `!dir`)
4. Copy `C:\Windows\System32\cmd.exe` or `powershell.exe` to your downloads and run from there
5. If your school has vscode installed just press ctrl shift \` and it might work
6. if your computer doesnt have bios lock follow something like [this](https://medium.com/@sauravraja/unlocking-administrative-access-a-simple-guide-to-bypassing-pc-restrictions-517a4f73a06f)

**Comment by Charlieis1nsan3** [comment]
👍

**Reply by Charlieis1nsan3** [reply]
blud he's talking about windows cmd prompt not chrome Linux terminal, or crosh :skull:
